docs/api/browser-window.md

```diff
@@ -392,10 +392,9 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       contain the layout of the document—without requiring scrolling. Enabling
       this will cause the `preferred-size-changed` event to be emitted on the
       `WebContents` when the preferred size changes. Default is `false`.
-  * `titleBarOverlay` Boolean (optional) -  On macOS, when using a frameless window in conjunction with
-    `win.setWindowButtonVisibility(true)` or using a `titleBarStyle` so that the traffic lights are visible,
-    this property enables the Window Controls Overlay [JavaScript APIs][overlay-javascript-apis] and
-    [CSS Environment Variables][overlay-css-env-vars].  Default is `false`.
+  * `titleBarOverlay` Boolean (optional) -  When using a frameless window in conjuction with `win.setWindowButtonVisibility(true)` on macOS or using a `titleBarStyle` so that the standard window controls ("traffic lights" on macOS) are visible, this property enables the Window Controls Overlay [JavaScript APIs][overlay-javascript-apis] and [CSS Environment Variables][overlay-css-env-vars]. Default is `false`.
+  * `overlayColor` String (optional) - The CSS color of the Window Controls Overlay when enabled on Windows. Default is the system color.
+  * `overlaySymbolColor` String (optional) - The CSS color of the symbols on the Window Controls Overlay when enabled on Windows. Default is the system color.
```

Comment:
On Windows there is already a thing named "overlay icon", "overlayColor" is probably too vague here. Maybe using a dumb name like "windowButtonColor"?

Reply:
How about `overlayButtonColor`? I think it would still be helpful to keep some cohesion between the button and symbol color parameter names in this way. 
This also keeps it in agreement with  the `titleBarOverlay` parameter naming. The alternative could be prepending `window` and/or `button` to `SymbolColor` but that may be more confusing (and possibly lengthy).

Reply:
How about we make `titleBarOverlay` a `Boolean | Object`.

* `false` === default, disabled
* `true` === enabled
* `{ color?: string; symbolColor?: string }` === enabled with customization

Reply:
I think that is a nice alternative. @zcbenz what are your thoughts?

Reply:
I'm good with any of the suggested alternatives above.

Reply:
Great, I went ahead and changed it to the boolean or object implementation in the commit "modify titlebar_overlay_ as boolean or object"